src/main/java/seedu/address/model/FlashCardTestModel.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+
+import seedu.address.model.flashcard.FlashCard;
+
+//@@author keiteo
+/**
+ * Creates a test model to contain relevant flashcards to test users.
+ */
+public class FlashCardTestModel {
+
+    private ArrayList<FlashCard> testList;
```

Comment:
You might want to consider having this variable be of type List<FlashCard> instead - it's better to program to interfaces where possible.